### DISCUSSION TITLE
migration-traps/

### DISCUSSION THREAD

**Discussion by giscus** [r-value/hugo-blog#53] [Announcements]
# migration-traps/

由于最近开始学东西了所以打算继续维护一下博客 但是想到 cnblogs 的不稳定性以及买了一个 rvalue.moe 的域名却一直闲置, 于是就有了重新构造一个博客的想法 这里记录一下

https://blog.rvalue.moe/migration-traps/

**Comment by r-value** [comment]
评论区已迁移到 Giscus (

**Comment by LoganQiu** [comment]
博主你现在还是使用 Shortcode 还是用 fenced code blocks？Hugo 现在官网上直接给出了后者的使用方法：https://gohugo.io/content-management/diagrams/#mermaid-diagrams
我现在一心想把 Typora 同时支持的 [flow](https://flowchart.js.org/) 和 [sequence](https://bramp.github.io/js-sequence-diagrams/) 也给捣鼓进 code blocks 里面一直不成功。。。
还有[数字电路: 锁存器/触发器的构造设计](https://blog.rvalue.moe/structure-and-design-of-latches-and-flip-flops/)这篇文章里的 img 可以用 Hugo 自带的 [GoAT](https://gohugo.io/content-management/diagrams/) 优雅实现，就是输入有些麻烦 :(

**Reply by r-value** [reply]
我已经相当长的时间没有 follow hugo 的更新了, 里面有些坑可能过时了...不过感谢提醒, 看上去支持 code fence 之后应该好用多了.  
至于数电里的 img 我其实大部分是偷懒引的外链 (目移
